internal/integration/provision/upgrade.go

```diff
@@ -224,8 +224,12 @@ func (suite *UpgradeSuite) TearDownSuite() {
 
 // setupCluster provisions source clusters and waits for health.
 func (suite *UpgradeSuite) setupCluster() {
-	shortNameHash := sha256.Sum256([]byte(suite.spec.ShortName))
-	clusterName := fmt.Sprintf("upgrade.%x", shortNameHash[:8])
+	defaultStateDir, err := clientconfig.GetTalosDirectory()
```

Comment:
relocate state dir to default talosctl location, allows to use `talosctl cluster destroy` to clean up